pages/design-tokens/typesetting/font-family.md

```diff
@@ -163,9 +165,9 @@ Role-based tokens set the font family value based on the _role_ the face plays i
 **Note:** It is possible to add **custom font metadata**, **custom font stacks**, and **custom font source files** in your USWDS settings. This documentation is coming soon. See the inline documentation in `_uswds-theme-typography` for more details.
```

Comment:
Slightly out of scope, but this file name is out of date.

```suggestion
**Note:** It is possible to add **custom font metadata**, **custom font stacks**, and **custom font source files** in your USWDS settings. This documentation is coming soon. See the inline documentation in `_settings-typography.scss` for more details.
```

Reply:
Related, *should* we create documentation for these? If not, maybe we can remove the coming soon line?